livnote/frontend/desktop/components/notes/setup/menuItems.ts

```
@@ -579,23 +579,23 @@ export function addTextSizeControls(container: HTMLElement, schema: Schema, view
 
     // 1. Check for explicit fontSize mark at cursor position
     const marks = $from.marks();
     if (schema.marks.fontSize) {
-        const fontSizeMark = schema.marks.fontSize.isInSet(marks);
-        if (fontSizeMark && fontSizeMark.attrs.size) {
-          // Use the explicit mark's size if it exists
-          fontSizeInput.value = fontSizeMark.attrs.size;
-          return;
-        }
+      const fontSizeMark = schema.marks.fontSize.isInSet(marks);
+      if (fontSizeMark && fontSizeMark.attrs.size) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)